### PR TITLE
Virtualize the covid-19 active case dataset to the webpage.

### PR DESCRIPTION
**Pull request by jiaxi312**
- Create a java servlet to process the data file.

- Virtualize the data using the google char api and display it to the web page.

- The screen shoot is https://screenshot.googleplex.com/sjAKpAXJip8.png

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/** When requested, returns the covid-19 active cases in each country as a JSON object */
+@WebServlet("/covid-19-data")
+public class COVID19DataServlet extends HttpServlet {
+
+  @Override
+  /** Reads the data from the local file */
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+      "/WEB-INF/country_wise_latest.csv"));
```
Comment (zixiyan) [inline portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java]: per https://screenshot.googleplex.com/nMLdzbk8fES,
use +4 spaces for indenting breaking lines.

File: portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    scanner.nextLine();
```
Comment (zixiyan) [inline portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java:23]: Add comments so others could know you want to skip the header.

File: portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java (hunk 1 above)
Comment (jiaxi312) [reply portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java]: Done!

File: portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java (hunk 2 above)
Comment (jiaxi312) [reply portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java:23]: Added!

Comment (zixiyan) [reply portfolio/src/main/java/com/google/sps/servlets/COVID19DataServlet.java:23]: Actually you want to skip the **header** line.

**Review by zixiyan — APPROVED** [review]
Approve to unblock

**Pull request merged**